tests/pos/i16458.scala

```diff
@@ -0,0 +1,37 @@
+def x = <div>FooBar</div><!-- /.modal-content -->
+
+package scala.xml {
```

Comment:
I copy-pasted and modifed code from tests/run/xml.scala. Without this, the test fails due to missing xml module.